ginivision/src/main/java/net/gini/android/vision/GiniVisionCoordinator.java

```diff
@@ -24,6 +25,10 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(GiniVisionCoordinator.class);
 
+    public static boolean shouldShowGiniVisionNoResultsScreen(final Document document) {
```

Comment:
Helper method which contains the rules for allowing the showing of our No Results Screen.